mackup/applications/forklift-3.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = ForkLift 3
+
+[configuration_files]
+Library/Preferences/com.binarynights.ForkLift-3.plist
```

Comment:
I would also add the ForkLift Favorites folder located at `~/Library/Application Support/ForkLift/Favorites`